packager/mpd/base/xml/xml_node.h

```diff
@@ -205,8 +205,11 @@ class RepresentationXmlNode : public RepresentationBaseXmlNode {
   /// Adds fields that are specific to VOD. This ignores @a media_info fields
   /// for Live.
   /// @param media_info is a MediaInfo with VOD information.
-  /// @param use_segment_list is a param that instructs the xml writer to use SegmentList instead of SegmentBase.
-  /// @param target_segment_duration is a param that provides the user give value for --segment_duration.
+  /// @param use_segment_list is a param that instructs the xml writer to
+  ///         use SegmentList instead of SegmentBase.
+  /// @param target_segment_duration is a param that provides the user
+  ///         given value for --segment_duration. This is only used when
+  ///         --use_segment_list is true.
```

Comment:
ditto. One extra space in both line 211 and 212.

I also suggest rephrasing the comments to something like:

target_segment_duration specifies the target duration of media segments. This is only used when `use_segment_list` is true.

(i.e. avoid using `--` in function parameter comments)